### DISCUSSION TITLE
Can't add user

### DISCUSSION THREAD

**Discussion by jdmcfadden-usx** [datalust/seq-tickets#2018] [Help]
Whenever I try to add a user now, I get the following error:

**The Active Directory principal search failed.**

It happens with any user I try to add, which suggests to me that my account may no longer have the authority it used to have. I did check my account in SEQ, however, and I am still listed as an administrator for the product.

**Comment by liammclennan** [comment]
Hi @jdmcfadden-usx 

This error indicates that the user that the Seq process is running as does not have sufficient permission to search active directory. You can configure Seq to run as a different user, or give the current user more permissions.

**Comment by jdmcfadden-usx** [comment]
Liam,

I talked to one of our network guys, and he couldn’t find a reason why our account can’t access ActiveDirectory:
***@***.***
I could put somebody from your team on a call with Justin. Otherwise, can you give him a way he could reproduce what the SEQ service does when it authenticates a new user? Is there logging for SEQ itself we could check to find out more about what’s going wrong? We’re dead in the water without some guidance.

Thanks.

[Shape, background pattern    Description automatically generated] Jim McFadden | Driver Experience support
U.S. Xpress
4080 Jenkins Road | Chattanooga, TN 37421
***@***.******@***.***>
423-667-9136
www.usxpress.com<https://urldefense.com/v3/__https:/link.edgepilot.com/s/22e78a68/u-0cKtsag0SquiP79xB-CA?u=http:**Awww.usxpress.com*__;Ly8v!!JywAMcrl3w!plOoywv15o-qYo4drLBjJNQbaDZ3nVAFOnJZlf-HS5HQ6Gnq4DH5zbH1OYYQqmkV$>




From: Liam McLennan ***@***.***>
Sent: Tuesday, November 14, 2023 6:54 PM
To: datalust/seq-tickets ***@***.***>
Cc: Jim McFadden ***@***.***>; Mention ***@***.***>
Subject: EXTERNAL: Re: [datalust/seq-tickets] Can't add user (Discussion #2018)


Hi @jdmcfadden-usx<https://urldefense.com/v3/__https:/github.com/jdmcfadden-usx__;!!JywAMcrl3w!8q_AjDVa6IRhOkvJgyXXfoUJQeRbAP4iHOoNKAFgxxMCPKxaHkwzxtkASP36vtcftBRt8e5QUCtPA9xtnK4r3UMeQXE$>

This error indicates that the user that the Seq process is running as does not have sufficient permission to search active directory. You can configure Seq to run as a different user, or give the current user more permissions.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/datalust/seq-tickets/discussions/2018*discussioncomment-7570903__;Iw!!JywAMcrl3w!8q_AjDVa6IRhOkvJgyXXfoUJQeRbAP4iHOoNKAFgxxMCPKxaHkwzxtkASP36vtcftBRt8e5QUCtPA9xtnK4ru7FVFXI$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AEZTNN7BPHR6UY2SNBYQZV3YEQAAZAVCNFSM6AAAAAA7LCHJQCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNZQHEYDG__;!!JywAMcrl3w!8q_AjDVa6IRhOkvJgyXXfoUJQeRbAP4iHOoNKAFgxxMCPKxaHkwzxtkASP36vtcftBRt8e5QUCtPA9xtnK4r9SaPPIA$>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>


Please Be Cautious: This email originated outside of U.S. Xpress or Total Transportation. Do not open attachments or click links unless you recognize the sender and know the content is safe.

**Reply by nblumhardt** [reply]
Hi @jdmcfadden-usx, thanks for the follow-up. We can jump on a call if needed, but the time zone overlap between us means it'll likely take a few days to set things up, so probably best to try to knock out the common causes of this issue first if we can.

Seq did change its default service account configuration a few versions back, from LocalSystem to a limited account called SeqDefaultInstance. You can check out which account Seq is using via the Services.msc applet.

If the account is the SeqDefaultInstance one, there's a good chance it won't have the privileges needed to interact with AD, and your best bet there is to reconfigure your Seq instance with a domain account.

LocalSystem does usually have the required permissions, as long as the Windows host machine does. If Seq is running as LocalSystem already, there are some configuration changes that you can try at the Seq end before worrying about machine permissions on the domain.

GitHub's probably not the best forum for going deeper on those details, so perhaps it's best to shift this thread to `support@datalust.co` from here?

Best regards,
Nick
